docs/contracts/v4/quickstart/03-swap.mdx

```diff
@@ -209,11 +214,13 @@ The sequence of these parameters must match the sequence of actions we defined e
 Now we can execute the swap using the Universal Router:
 
 ```solidity
+bytes[] memory inputs = new bytes[](1);
+
 // Combine actions and params into inputs
 inputs[0] = abi.encode(actions, params);
 
 // Execute the swap
-router.execute(commands, inputs, deadline);
+router.execute(commands, inputs, block.timestamp);
```

Comment:
its actually not good practice to use block.timestamp for deadline -- the transaction will likely fail

if we we want to be accurate we should define `uint256 deadline = block.timestamp + 20;` in both code blocks